### PR TITLE
Add RFC for approver requirements

### PR DESCRIPTION
**Pull request by ctlong**
cc @ameowlia

### CODE REVIEW

**Review by dsboulder — APPROVED** [review]

**Review by rkoster — APPROVED** [review]

**Comment by dsboulder** [comment]
I'm happy with this! Please tag the other leads for feedback and start the 1 week review process.

**Comment by ctlong** [comment]
👋  Hi @jochenehret @Gerg @georgethebeatle @gcapizzi @beyhan @dmikusa-pivotal @ekcasey @ryanmoran @pivotal-marcela-campo @paulcwarren.

👀  This RFC will affect the conditions under which approvers are nominated to your WGs.

➡️  Could y'all please have a look through this document and let us know what you think.

**Comment by ctlong** [comment]
This starts the 1 week review process 🎉

**Comment by ryanmoran** [comment]
I'm interested to hear what the TOC thinks about this PR and the broader [ROLES](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md) document given the Paketo WGs "special status" as a separate project with its own governing structure. Do we need to have an explicit carveout for some of these specifications or is that covered elsewhere?

**Review by jochenehret — APPROVED** [review]
Looks good... not sure however how strictly we want to apply the "complete 20 of the following..." rule.

File: toc/rfc/rfc-draft-approver-requirements.md
Comment (Gerg) [inline toc/rfc/rfc-draft-approver-requirements.md:64]: Should we break the process for people moving between roles into a separate RFC? Keep this one solely focused on the criteria and another one could talk about the mechanics of people gaining/losing roles within working groups.

I can take a pass at writing that RFC.

Comment (Gerg) [reply toc/rfc/rfc-draft-approver-requirements.md:64]: Drafted the RFC here: https://github.com/cloudfoundry/community/pull/248

Comment (stephanme) [reply toc/rfc/rfc-draft-approver-requirements.md:64]: The source-of-truth should be the working-group charter document (the yaml part). 
This could be improved: ... they can then open a PR
against the community repo adding their name to the list of approvers in a
[Working Group charter](https://github.com/cloudfoundry/community/tree/main/toc/working-groups) document.

Editing cloudfoundry.yml directly as proposed in #248 is rather a workaround because the translation of WG charters to cloudfoundry.yml is not yet automated. Not sure if the workaround is worth an RFC.

Comment (ameowlia) [reply toc/rfc/rfc-draft-approver-requirements.md]: Sounds good. I removed that section.

**Review by loewenstein — APPROVED** [review]

**Comment by ameowlia** [comment]
Hi @ryanmoran,

> I'm interested to hear what the TOC thinks about this PR and the broader [ROLES](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md) document given the Paketo WGs "special status" as a separate project with its own governing structure. Do we need to have an explicit carveout for some of these specifications or is that covered elsewhere?

The TOC chatted about this today. We decided not to make an _explicit_ carveout for Paketo in this RFC. We agreed that because Paketo has their own [governance](https://github.com/paketo-buildpacks/community/blob/main/GOVERNANCE.md), that this is an _implicit_ carveout for many things, including approver requirements.

**Review by stephanme — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Review by Gerg — APPROVED** [review]

**Review by emalm — APPROVED** [review]

**Comment by emalm** [comment]
This RFC now has unanimous approval from @cloudfoundry/toc and no outstanding discussion threads, so it is cleared to accept. I'll plan to merge it by the end of the week, unless another TOC member can take care of it first.

Note that acceptance of this RFC will then require a change to the ROLES.md document that describes the Approver criteria, which should come in a separate PR to be reviewed for consistency with this RFC's text.

**Review by beyhan — APPROVED** [review]

**Review by pivotal-marcela-campo — APPROVED** [review]

**Pull request merged**
